src/www/ui/template/user_edit.html.twig

```diff
@@ -65,13 +66,26 @@
       </tr>
       {% endif %}
     {% endif %}
-
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>Toggle Password Visibility</title>
```

Comment:
The `<meta>` and `<title>` should be part of `<head>` and not of `<body>`. Please remove from here.